settings.gradle.kts

```diff
@@ -68,7 +68,6 @@ include(
 
     ":plugins:base",
     ":plugins:base:frontend",
-    ":plugins:base:search-component",
```

Comment:
I'm pretty sure that this subproject is unused? I couldn't find any usages, and it was empty.

Reply:
It might've existed for what now resides in `frontend/src/main/components/search`, but it indeed looks to be unused as of this moment :+1: